code/game/area/areas/ruins/lavaland.dm

```diff
@@ -69,12 +69,12 @@
 
 /area/ruin/powered/kinggoat_arena //yogs start
 	name = "King Goat Arena"
-	noteleport = TRUE
+	area_flags = NOTELEPORT
```

Comment:
Does parent have any flags that they should keep?
```suggestion
	area_flags = parent_type::area_flags | NOTELEPORT
```

Reply:
wait, you can do that?

Reply:
None of the other area flags are used outside lavaland generation so no but I can do this anyways